soes/esc_coe.c

```diff
@@ -872,7 +872,9 @@ static void SDO_download (void)
          }
          else
          {
-            if (access == ATYPE_RWpre)
+            if ((access == ATYPE_RWpre) ||
```

Comment:
This is testing if write was allowed in any state? Would (access & ATYPE_WO) be equivalent and perhaps clearer?

Reply:
I've swapped the logic